scripts/readme-template.md

```diff
@@ -0,0 +1,59 @@
+# n-heroku-tools (nht) [![CircleCI](https://circleci.com/gh/Financial-Times/n-heroku-tools.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://circleci.com/gh/Financial-Times/n-heroku-tools)
+
+This library is a command line tool that orchestrates [Heroku](https://www.heroku.com/) and [Amazon S3](https://aws.amazon.com/s3/) deployments for [Next](https://github.com/Financial-Times/next/wiki), based on configurations in the [Next service registry](https://next-registry.ft.com/v2/) and [Vault](https://www.vaultproject.io/).
```

Comment:
Are the Amazon S3 deployments for hashed assets and static websites only?

I.e. I am guessing they do not include S3-hosting for Serverless applications?

Reply:
correct! it simply uploads files to S3 but it wont run serverless applications.